src/app/components/Footer/Footer.scss

```diff
@@ -0,0 +1,177 @@
+@import '../../../variables';
+
+footer {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    padding-top: 460px;
+    margin-left: 10px;
+
+    .main-interno {
+        display: flex;
+        flex-direction: row;
+        align-items: flex-start;
+        justify-content: center;
+        width: 100%;
+        height: 100%;
+        gap: 11%;
```

Comment:
flex-direction: row is a default style, so you can remove this line
also, is it really a good idea to have such gap in %?